docs/configuration.rst

```diff
@@ -190,6 +190,15 @@ Here is a more complex example:
        *.ignore-trailing-spaces.yaml
        ascii-art/*
 
+You can also use the ``.gitignore`` (or any file or files) directly through:
+
+.. code-block:: yaml
+
+ # For all rules
+ ignore-from-file: [.gitignore, .yamlignore] # or: ignore-from-file: .gitignore
+
+.. note:: However, is mutually exclusive with the ``ignore`` key.
```

Comment:
I´m not a native English speaker, so please contradict me if I´m wrong. But I believe `this is` would be more correct than `is`?